app/src/matrix_transform.c

```diff
@@ -33,7 +36,12 @@ uint32_t zmk_matrix_transform_row_column_to_position(uint32_t row, uint32_t colu
     matrix_index = (row * ZMK_MATRIX_COLS) + column;
 
 #ifdef ZMK_KEYMAP_TRANSFORM_NODE
-    return transform[matrix_index];
+    if (matrix_index >= ARRAY_SIZE(transform))
+        return -EINVAL;
```

Comment:
I'm not a fan of `if` blocks without braces, but clang-format should automatically change something like

```c
if (error)
    LOG_ERR("Just did a bad thing");
    return -EINVAL;
```
to
```c
if (error)
    LOG_ERR("Just did a bad thing");
return -EINVAL;
```

and make that error more obvious. I see we already do this in several places throughout the codebase too, so if Pete's okay with this, I won't object.

Reply:
DONE, I added braces.